src/frontend/screens/Login/components/Runner/index.tsx

```diff
@@ -26,40 +26,54 @@ export default function Runner(props: RunnerProps) {
     //window.localStorage.clear()
   }
   return (
-    <div className={`runnerWrapper ${props.class}`}>
-      <div>{props.icon()}</div>
-      {props.isLoggedIn && (
-        <div className="userData">
-          <span>
-            {String(props.user).slice(0, maxNameLength) +
-              (String(props.user).length > maxNameLength ? '...' : '')}
-          </span>
-        </div>
-      )}
-      <div>
-        {!props.isLoggedIn ? (
-          <Link to={props.loginUrl}>
-            <div className="runnerLogin">{t('button.login', 'Login')}</div>
-          </Link>
-        ) : (
-          <div
-            className="runnerLogin logged"
-            onClick={() => {
-              handleLogout()
-            }}
-          >
-            {t('userselector.logout', 'Logout')}
+    <>
+      <div className={`runnerWrapper ${props.class}`}>
+        <div className={`runnerIcon ${props.class}`}>{props.icon()}</div>
+        {props.isLoggedIn && (
+          <div className="userData">
+            <span>
+              {String(props.user).slice(0, maxNameLength) +
+                (String(props.user).length > maxNameLength ? '...' : '')}
+            </span>
           </div>
         )}
+        <div className="runnerButtons">
+          {!props.isLoggedIn ? (
+            <Link to={props.loginUrl}>
+              <div className="runnerLogin">{`${
+                props.class === 'epic'
+                  ? props.class + ' Games Login'
+                  : props.class + ' Login'
+              }`}</div>
```

Comment:
let's make this a more explicit if/else, this is not translation friendly:

```
let buttonText = ''

if (props.class === 'epic') {
  buttonText = t('epic_login', 'Epic Games Login')
} else {
  buttonText = t('gog_login', 'GOG Login')
}
```

And then `<Link to...>{buttonText}</Link>`

Then running `yarn i18next` will update the locale files.